tests/test_persistence.py

```diff
@@ -335,6 +335,23 @@ def good_pickle_files(user_data, chat_data, bot_data, conversations):
         os.remove(name)
 
 
+@pytest.fixture(scope='function')
+def pickle_files_wo_bot_data(user_data, chat_data, conversations):
+    all = {'user_data': user_data, 'chat_data': chat_data, 'conversations': conversations}
+    with open('pickletest_user_data', 'wb') as f:
```

Comment:
Better use `tempfile` module here. You can never know when you'll have same tests running in parallel on the same machine.

Reply:
Don't know about tempfile. But the fixture is basically just copy-paste from the others. So I'd rather open an issue to change them all and leave this as is for now, if that's okay with you